src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -103,6 +103,7 @@ object BitSet extends BitSetFactory[BitSet] {
     else new BitSetN(elems)
   }
 
+  @SerialVersionUID(2260107458435649300L)
```

Comment:
@SethTisue - Doubt it.  Not sure where that came from.  I'll check it out later when I have time.